src/h5web/providers/Provider.tsx

```diff
@@ -51,7 +51,11 @@ function Provider(props: Props) {
         filepath: api.filepath,
         entitiesStore,
         valuesStore,
-        cancel: () => api.cancel(),
+        cancelValues: () => api.cancelValues(),
+        evictCancelledValues: () => {
+          api.cancelledValueParams.forEach((key) => valuesStore.evict(key));
+          api.cancelledValueParams.clear();
+        },
```

Comment:
The solution works in two stages:

1. When the user clicks on "Cancel", the `cancelValues` method is called. This cancels the requests and stores the params that were used to make them in a `Set` named `cancelledValueParams`.
2. The error boundary catches the "Request cancelled" error, then calls its `onError` handler. The handler then calls `evictCancelledValues`, which reads the params of the requests that have been cancelled and evict the corresponding cache entries from the `valuesStore`.